src/ppom-storage.ts

```diff
@@ -0,0 +1,204 @@
+import { calculateSHA256 } from './crypto-utils';
+
+type FileInfo = {
```

Comment:
Nit: We don't have to do this right now, but it would be nice to provide some documentation for these types, to describe what they do and what their purpose is. This way consumers can see that documentation when hovering over the type in their editor and also so that they would appear in generated documentation.